build/docker/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apk add --no-cache \
     make \
     musl-dev \
     jq \
+    py3-setuptools \
```

Comment:
i think this requires python3 🤔  https://pkgs.alpinelinux.org/package/edge/main/x86/py3-setuptools ? maybe i am wrong 🤷🏻‍♀️

Reply:
its already installed by default